engine/src/flutter/shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -1942,10 +1970,13 @@ void PlatformViewAndroidJNIImpl::onDisplayPlatformView2(
       mutators_stack.Begin();
   while (iter != mutators_stack.End()) {
```

Comment:
A note for @gmackall , we should refactor this logic to share the mutator encoding between the two platform view JNI methods.

Reply:
Should this and the 4x4 vs 3x3 issue be combined or separate?

Reply:
I'll leave this to @gmackall to file if necessary.